packages/app/components/strategies/constants.tsx

```diff
@@ -0,0 +1,80 @@
+import { FREQUENCY_OPTIONS } from "@/models/stack";
+import { GNO, USDC, WBTC, WETH, WXDAI } from "@/models/token";
+
+export const FREQUENCY_LABEL = {
+  [FREQUENCY_OPTIONS.hour]: "hourly",
+  [FREQUENCY_OPTIONS.day]: "daily",
+  [FREQUENCY_OPTIONS.week]: "weekly",
+  [FREQUENCY_OPTIONS.month]: "monthly",
+};
+
+export const STRATEGY_CATEGORIES = {
+  /** 
+   * TBD in STK-190
+   * @see https://linear.app/swaprhq/issue/STK-190/add-strategies-for-you-strategies-tab
+   * @TODO uncomment the following lines when
+   * creating the "Strategies for you" section
+   * and simply fill the `strategies` array
+   * following the same data structure than `popular`
+  suggested: {
+    label: () => (
+      <>
+        <Icon className="mr-1.5" name="sparkles" size={14} /> Strategies for you
+      </>
+    ),
+    strategies: [
+      
+    ],
+  },
+  */
```

Comment:
I don't think that we should have this comment here, linear task is enough I think,

Reply:
💯